src/WebJobs.Script.WebHost/Diagnostics/Sanitizer.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.WebJobs.Logging
     internal static class Sanitizer
     {
         private const string SecretReplacement = "[Hidden Credential]";
-        private static readonly char[] ValueTerminators = new char[] { '<', '"' };
+        private static readonly char[] ValueTerminators = new char[] { '<', '"', '\'' };
```

Comment:
How does this fix the issue? Can you add a test to show it? I thought the problem was that one of our `CredentialTokens` is `Token=` -- which unfortunately is part of the assembly identifier (`PublicKeyToken=`). We want to make sure we continue sanitizing anything with `Token=` but not with `PublicKeyToken=`.

Reply:
Example from #2188, this method was removing rest of the stack trace i.e till end of the string as  char ' was missing from the set of Value terminators. Do we want to keep "PublicKeyToken=" ?

Reply:
Added tests and updated sanitize method to exclude known keywords